core/arch/arm/kernel/secstor_ta.c

```diff
@@ -69,18 +69,10 @@ static void secstor_ta_close(struct user_ta_store_handle *h)
 	tee_tadb_ta_close(ta);
 }
 
-static struct user_ta_store_ops ops = {
+TEE_TA_REGISTER_TA_STORE(4) = {
```

Comment:
Any reason for choosing 2, 4 and 9 as the priorities instead of 5, 9 and 10?

Reply:
They sorted alphabetically alphabetically I had to change it to single letter priories and while I was at it I thought that this allocation was a bit more flexible. I'll add something about this in the commit message.

Reply:
Ah, yes, it is indeed very important to be aware that priority is not numeric but lexicographic. Can you please add a comment next to the definition of the `TEE_TA_REGISTER_TA_STORE()` macro? I'm fine with the new values as such.

Reply:
I'll add a comment